bridge/processor/span.go

```diff
@@ -170,6 +280,25 @@ func (sp *SpanProcessor) getLastSpan() (*types.Span, error) {
 	return &lastSpan.Span, nil
 }
 
+// get span by id
+func (sp *SpanProcessor) getSpanById(id uint64) (*types.Span, error) {
+	// fetch latest start block from heimdall via rest query
```

Comment:
Are we fetching the latest start block here ?

Reply:
Fixed wrong comment.